README.md

```diff
@@ -5,5 +5,6 @@ Example code for using SigOpt.
 - [ipython-notebook-example](https://github.com/sigopt/sigopt-examples/tree/master/ipython-notebook-example): Simple example of using SigOpt to optimize a 2D function with plots and comparisons.
 - [classifier](https://github.com/sigopt/sigopt-examples/tree/master/classifier): Using SigOpt to tune a machine learning classifier in Python.
 - [parallel](https://github.com/sigopt/sigopt-examples/tree/master/parallel): Examples of running SigOpt from multiple parallel processes in Python.
+- [text_classifier](https://github.com/sigopt/sigopt-examples/tree/master/text_classifier): Example using SigOpt to tune a text classifier in Python.
```

Comment:
Let's bump this to #2. Best thing to read after looking over notebook for quick intro imo.